docs/proposals/kubeedgeadm-scope.md

```diff
@@ -0,0 +1,255 @@
+
+### kubeedge --help or kubedge
+
+```
+ubuntu# kubedge --help or ubuntu# kubedge
+Kubeedge
+
+    ┌──────────────────────────────────────────────────────────┐
+    │ KUBEEDGE                                                 │
+    │ Easily bootstrap KubeEdge cluster                        │
+    │                                                          │
+    │ Please give us feedback at:                              │
+    │ https://github.com/kubeedge/kubeedge/issues              │
+    └──────────────────────────────────────────────────────────┘
+
+Example usage:
+
+    Create a two-machine cluster with one cloud node
+    (which controls the edge cluster), and one edge node
+    (where native containerized application, in the form of
+    pods and deployments run), connects to devices.
+
+    ┌──────────────────────────────────────────────────────────┐
+    │ On the first machine:                                    │
+    ├──────────────────────────────────────────────────────────┤
+    │ cloud-node# kubeedge cloud init <arguments>              │
+    └──────────────────────────────────────────────────────────┘
+
+    ┌──────────────────────────────────────────────────────────┐
+    │ On the second machine:                                   │
+    ├──────────────────────────────────────────────────────────┤
+    │ edge-node# kubeedge node join <arguments>                │
+    └──────────────────────────────────────────────────────────┘
+
+    You can then repeat the second step on as many other machines as you like.
+
+Usage:
+  kubeedge [command]
+
+Available Commands:
+  cloud       Cloud component command option for KubeEdge
+  node        Edge component command option for KubeEdge
+  version     Displays KubeEdge release version and code commit id.
+
+Flags:
+  -h, --help   help for kubeedge
+
+Use "kubeedge [command] --help" for more information about a command.
+ubuntu#
+```
+
+### kubeedge cloud --help
+
+```
+ubuntu# kubeedge cloud --help
+<Apt description to be added while implementation>
+
+Usage:
+  kubeedge cloud [command]
+
+Example usage:
+<Apt information to be added while implementation>
+
+Available Commands:
+  init        Bootstraps cloud component. Checks and install (if required) the pre-requisites.
+  reset       Teardowns cloud component.
+
+Flags:
+  -h, --help   help for cloud
+
+Use "kubeedge cloud [command] --help" for more information about a command.
+```
+
+### kubeedge cloud init --help
+
+```
+kubeedge cloud init --help
+<Apt description to be added while implementation>
+
+Usage:
+  kubeedge cloud init [flags]
+
+Example usage:
+<Apt information to be added while implementation>
+
+Flags:
+  -h, --help                        help for init
+      --kubeedge-version   string   use this key to download and use the required KubeEdge version (Optional, default will be Latest)
+      --kubernetes-version string   use this key to download and use the required Kubernetes version (Optional, default will be Latest)
```

Comment:
we need to double check if all latest versions of docker / k8s work well. there may be cases where k8s latest version could work on a specific docker version. just curious about this. else , we could enforce stable versions as defaults which have been tested by community for compatibility.

Reply:
Yes, We will take of the supported and stable version of docker and K8S